manifests/kubernetes/base/deployment.yaml

```diff
@@ -115,6 +117,10 @@ spec:
         hostPath:
           path: /proc
           type: Directory
+      - hostPath: 
```

Comment:
this is only needed for kind cluster running on ubuntu runner
@jichenjc can you help to make this into a kustomization?